src/main/java/com/redhat/cloud/policies/app/EnvironmentFlags.java

```diff
@@ -0,0 +1,36 @@
+package com.redhat.cloud.policies.app;
+
+import io.quarkus.runtime.Startup;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.jboss.logging.Logger;
+
+import javax.annotation.PostConstruct;
+import java.lang.reflect.Field;
+
+@Startup
+public class EnvironmentFlags {
+
+    private static final Logger LOG = Logger.getLogger(EnvironmentFlags.class);
+
+    @PostConstruct
+    public void init() {
+        try {
+            LOG.info("Environment flags in use:");
+            for (Field field: EnvironmentFlags.class.getDeclaredFields()) {
+                ConfigProperty configAnnotation = field.getAnnotation(ConfigProperty.class);
+                if (configAnnotation != null) {
+                    LOG.infof("  - %s: %s", configAnnotation.name(), field.get(this));
+                }
+            }
+        } catch (IllegalAccessException iae) {
+            throw new RuntimeException("Error getting flags while starting.", iae);
+        }
+    }
+
+    @ConfigProperty(name = "flag.is-fedramp", defaultValue = "false")
+    Boolean isFedramp;
```

Comment:
Do we need this new config key or could we use [that](https://github.com/RedHatInsights/notifications-backend/blob/1af499740a5478949819353b025851ab347102bf/.rhcicd/clowdapp-backend.yaml#L76-L77) instead (the value is set by Clowder)?

Reply:
Ah, we could use that one. Do you know what value would take the environment on fedramp ?

Reply:
I'm not sure yet since Clowder hasn't been used to deploy fedramp until now.

We can merge this PR as-is and simplify it later when the Clowder migration is over.